docs/nodes/keys.mdx

```diff
@@ -0,0 +1,243 @@
+---
+sidebar_label: Keys
```

Comment:
Check the latest main, we have renamed this file to `[celestia-node-key](https://docs.celestia.org/developers/celestia-node-key)`

I'd update this PR to be based on the updates inside the new file name

Reply:
should be good in my last push, updated with the tabs i made on the new page